sbt-plugin/src/sbt-test/cross-version/2.13/src/main/scala/sbttest/Lib.scala

```diff
@@ -0,0 +1,11 @@
+package sbttest.noDOM
```

Comment:
Perhaps use the opportunity to rename this package. The test doesn't have anything to do with DOM or noDOM anymore.

(same in the other sources, as well as in the 2.11 sources)